src/push-manifest.ts

```diff
@@ -209,18 +211,26 @@ export class AddPushManifest extends AsyncTransformStream<File, File> {
           this.analyzer, absoluteFragmentUrl, fragmentIgnoreUrls);
     }
 
-    if (this.prefix) {
-      const updated: any = {};
-      for (const source of Object.keys(pushManifest)) {
-        const targets: any = {};
-        for (const target of Object.keys(pushManifest[source])) {
-          targets[this.prefix + target] = pushManifest[source][target];
-        }
-        updated[this.prefix + source] = targets;
+    // The URLs we got may be absolute or relative depending on how they were
+    // declared in the source. This will normalize them to relative by stripping
+    // any leading slash.
+    //
+    // TODO Decide whether they should really be relative or absolute. Relative
+    // was chosen here only because most links were already relative so it was
+    // a smaller change, but
+    // https://github.com/GoogleChrome/http2-push-manifest actually shows
+    // relative for the keys and absolute for the values.
+    const normalize = (p: string) =>
+        path.posix.join(this.basePath, p).replace(/^\/+/, '');
+
+    const normalized: any = {};
```

Comment:
Actually, turns out there was a type defined! We just weren't using it for some reason.